jwk.go

```diff
@@ -126,6 +132,16 @@ func (k *JSONWebKey) UnmarshalJSON(data []byte) (err error) {
 		}
 	case "oct":
 		key, err = raw.symmetricKey()
+	case "OKP":
+		if raw.Crv == "Ed25519" {
```

Comment:
Could add a check here that X is present, as it must always be present for both private and public keys:
```
if raw.Crv == "Ed25519 && raw.X != "" {
        // parse 
} else {
        // return error
}
```